runtime/executor/src/main/java/edu/snu/nemo/runtime/executor/datatransfer/OutputCollectorImpl.java

```diff
@@ -55,14 +66,20 @@ public void emit(final O output) {
       emit((O) output);
     } else {
       // Note that String#hashCode() can be cached, thus accessing additional output queues can be fast.
-      final List<Object> dataElements = this.additionalTagElementsMap.get(dstVertexId);
-      if (dataElements == null) {
```

Comment:
Why remove this checker?

Reply:
Because the checking is done inside `getAdditionalTaggedDataFromTag` 😄